### DISCUSSION TITLE
centos6.9 .accesslog error report

### DISCUSSION THREAD

**Discussion by crspecter** [juicedata/juicefs#4079] [General]
**What happened**:
**1.In centos6.9  access fs, the .accesslog report below**:
2023.09.21 13:57:20.307112 [uid:0,gid:0,pid:32700] lookup (1,x86_64): no such file or directory <0.000593>
2023.09.21 13:57:20.307672 [uid:0,gid:0,pid:32701] lookup (1,x86_64): no such file or directory <0.000519>
2023.09.21 13:57:20.308205 [uid:0,gid:0,pid:32700] lookup (1,libselinux.so.1): no such file or directory <0.000499>
2023.09.21 13:57:20.308701 [uid:0,gid:0,pid:32701] lookup (1,libdl.so.2): no such file or directory <0.000459>
2023.09.21 13:57:20.309313 [uid:0,gid:0,pid:32700] lookup (1,tls): no such file or directory <0.000569>
2023.09.21 13:57:20.309821 [uid:0,gid:0,pid:32700] lookup (1,tls): no such file or directory <0.000457>
2023.09.21 13:57:20.310248 [uid:0,gid:0,pid:32701] lookup (1,tls): no such file or directory <0.000387>
2023.09.21 13:57:20.310762 [uid:0,gid:0,pid:32700] lookup (1,x86_64): no such file or directory <0.000476>
2023.09.21 13:57:20.311314 [uid:0,gid:0,pid:32701] lookup (1,tls): no such file or directory <0.000510>
2023.09.21 13:57:20.311922 [uid:0,gid:0,pid:32700] lookup (1,libc.so.6): no such file or directory <0.000566>
2023.09.21 13:57:20.312507 [uid:0,gid:0,pid:32701] lookup (1,x86_64): no such file or directory <0.000540>
2023.09.21 13:57:20.313037 [uid:0,gid:0,pid:32700] lookup (1,tls): no such file or directory <0.000488>
2023.09.21 13:57:20.313604 [uid:0,gid:0,pid:32701] lookup (1,libm.so.6): no such file or directory <0.000524>
2023.09.21 13:57:20.314084 [uid:0,gid:0,pid:32700] lookup (1,tls): no such file or directory <0.000446>
2.use "df -h" can't show up the mountpoint
!951 # df -h
Filesystem      Size  Used Avail Use% Mounted on
/dev/sda3       557G  366G  191G  66% /
devtmpfs         94G     0   94G   0% /dev
tmpfs            94G   12K   94G   1% /dev/shm
tmpfs            94G  1.7G   93G   2% /run
tmpfs            94G     0   94G   0% /sys/fs/cgroup
/dev/sda2      1014M  119M  896M  12% /boot
/dev/sda1       500M   12M  489M   3% /boot/efi
/dev/sdb1        11T  8.2T  2.4T  78% /data
tmpfs            19G     0   19G   0% /run/user/1100
tmpfs            19G     0   19G   0% /run/user/0
JuiceFS:binlog  1.0P   52K  1.0P   1% /data/binlog  **This line does not appear in centos6.9
**What you expected to happen**:**
no error report and mountpoint can be show  when run cmd  'df -h'
**How to reproduce it (as minimally and precisely as possible)**:
  
**Anything else we need to know?**

**Environment**:
- JuiceFS version (use `juicefs --version`) or Hadoop Java SDK version:
  v1.0.4

- Cloud provider or hardware configuration running JuiceFS:
   hardware configuration running JuiceFS
- OS (e.g `cat /etc/os-release`): 
  CentOS release 6.9 (Final)
- Kernel (e.g. `uname -a`):
  Linux BJ-YZ-2FM1-B04-DB-12-1 2.6.32-696.el6.x86_64 #1 SMP Tue Mar 21 19:29:05 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
- Object storage (cloud provider and region, or self maintained):
   Cos
- Metadata engine info (version, cloud provider managed or self maintained):
   mysql (self maintain)
- Network connectivity (JuiceFS to metadata engine, JuiceFS to object storage):
- Others:

**Comment by SandyXSD** [comment]
The errors in .accesslog are normal. This line:
`2023.09.21 13:57:20.307112 [uid:0,gid:0,pid:32700] lookup (1,x86_64): no such file or directory <0.000593>`
says that the process with pid 32700 was trying to find a file named `x86_64` under the JuiceFS mount point, but can't find it.

**Comment by crspecter** [comment]
how about these lines "xx.so" , eg: libc.so.6 no such file or directory， further  why  “df - h” dose not  return the filesystem mountpoint

**Comment by SandyXSD** [comment]
> how about these lines "xx.so" , eg: libc.so.6 no such file or directory

It's application behavior, so you'd better check what the process accessing JuiceFS wants to do. These errors have nothing to do with the JuiceFS itself.

> further why “df - h” dose not return the filesystem mountpoint

Not sure about this issue. Since JuiceFS is not well tested on CentOS 6, I recommend you trying CentOS 7 or higher.
